### PR TITLE
Reduce startup time.

### PR DESCRIPTION
**Pull request by dmaclach**
Move file scope constant that is initialized at startup to function scope to avoid startup hit.

### CODE REVIEW

**Comment by pinterest-ios** [comment]
🚫 CI failed with [log](https://gist.githubusercontent.com/pinterest-ios/67e08d2819a8ed11ebb39f2c2d04a649/raw/45b13b917dfd78ddc02b0184997816bf73eebce3/log.txt)

**Comment by maicki** [comment]
The CI failure is not due to changes in this PR. @dmaclach would you mind to rebase from master to restart the CI. Thanks!

File: Source/ASDisplayNode.mm [hunk 1]
```diff
@@ -275,6 +274,7 @@ - (void)_staticInitialize
 
 - (void)_initializeInstance
 {
+  NSInteger const ASDefaultDrawingPriority = ASDefaultTransactionPriority;
```
[the remark this answers is not in the document]
Comment (dmaclach) [reply Source/ASDisplayNode.mm]: Done

**Comment by dmaclach** [comment]
Can somebody merge?

**Comment by dmaclach** [comment]
Thank you kindly!

**Pull request merged**